client/test_contracts/__init__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2015-2020 Clearmatics Technologies Ltd
```

Comment:
Could you please rebase on top of `develop` to make sure these are updated with the 2021 copyright mention and update any new header you added in this PR?

Reply:
should be rebased now.

Reply:
Thanks